apps/rush-lib/src/data/test/repo/rush.json

```diff
@@ -17,6 +17,12 @@
     "sampleEmail": "mrexample@contoso.com"
   },
 
+  "rushHooks": {
+    "postBuild": [
```

Comment:
Changed to postRushBuild

---
In reply to: [117142525](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r117142525) [](ancestors = 117142525,117142380)